extensions/cornerstone-dicom-sr/src/utils/addMeasurement.ts

```diff
@@ -61,6 +68,7 @@ export default function addMeasurement(
         TrackingUniqueIdentifier: measurementData.TrackingUniqueIdentifier,
         renderableData: measurementData.renderableData,
       },
+      frameNumber: frameNumber,
```

Comment:
we should not stick the `frameNumber` here, it is not a `data` per se. can't we just infer it from the referencedImageId? seeing if it has `frames/...` at the end?

Reply:
This is to conform with the changes we've made to dcmjs - cornerstone3D adapter: https://github.com/dcmjs-org/dcmjs/pull/317